src/ORM/Behavior/TranslateBehavior.php

```diff
@@ -247,11 +248,11 @@ public function beforeFind(Event $event, Query $query, $options)
      * in the database too.
      *
      * @param \Cake\Event\Event $event The beforeSave event that was fired
-     * @param \Cake\ORM\Entity $entity The entity that is going to be saved
+     * @param \Cake\Datasource\EntityInterface $entity The entity that is going to be saved
      * @param \ArrayObject $options the options passed to the save method
      * @return void
      */
-    public function beforeSave(Event $event, Entity $entity, ArrayObject $options)
+    public function beforeSave(Event $event, EntityInterface $entity, ArrayObject $options)
```

Comment:
We should avoid such type hinting changes on master in future. This change has caused strict error in [ShadowTranslate](https://github.com/AD7six/cakephp-shadow-translate/issues/17) which extends this class.